### PR TITLE
add cards task

### PR DESCRIPTION
**Pull request by AnoriFrell**
[DEMO LINK](https://AnoriFrell.github.io/layout_product-cards/)
 [TEST REPORT LINK](https://AnoriFrell.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

**Review by lerastarynets — CHANGES_REQUESTED** [review]
Pls try to pass all tests from backstop before asking for review again)

File: src/index.html [hunk 1]
```diff
@@ -5,9 +5,53 @@
...
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="container">
+      <div class="container__card card"
+          data-qa="card">
+        <img class="card__picture picture"
+          src="images/imac.jpeg"
+          alt="Apple A1419 iMac 27'' Retina 5K Monoblock"
+        >
```
Comment (volodymyr-soltys97) [inline src/index.html:20]: Need to check and fix it everywhere
image

File: src/index.html [hunk 2]
```diff
@@ -5,9 +5,53 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&   display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="container">
+      <div class="container__card card"
+          data-qa="card">
```
Comment (volodymyr-soltys97) [inline src/index.html:16]:
```suggestion
      <div class="container__card card" data-qa="card">
```

File: src/index.html [hunk 3]
```diff
@@ -5,9 +5,53 @@
...
+        <div class="price">
+          <p class="price__text">Price:</p>
+          <p class="price__number">$2,199</p>
+        </div>
+
+        <a
+          data-qa="hover"
+          href="#"
+          class="card__button button">
+          Buy
```
Comment (volodymyr-soltys97) [inline src/index.html:52]:
```suggestion
        <a
          data-qa="hover"
          href="#"
          class="card__button button"
        >
          Buy
```

File: src/styles/blocks/button.scss [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  width: 164px;
+  height: 38px;
+  color: #fff;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  border: 1px solid #00acdc;
+
+  &:hover {
+    color: #00acdc;
```
Comment (volodymyr-soltys97) [inline src/styles/blocks/button.scss:15]: Create a variables for repeating colors

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
This part of the task is incomplete
image
image

File: src/index.html [hunk 5]
```diff
@@ -5,9 +5,66 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link
+    rel="preconnect"
+    href="https://fonts.googleapis.com"
+    >
```
Comment (volodymyr-soltys97) [inline src/index.html:11]:
```suggestion
    <link rel="preconnect" href="https://fonts.googleapis.com">
```

File: src/styles/blocks/button.scss (hunk 4 above)
Comment (volodymyr-soltys97) [reply src/styles/blocks/button.scss:15]: Still not fixed

File: src/index.html (hunk 5 above)
Comment (volodymyr-soltys97) [reply src/index.html:11]: Still not fixed from the previous review, need to fix it

File: src/styles/blocks/button.scss (hunk 4 above)
Comment (volodymyr-soltys97) [reply src/styles/blocks/button.scss:15]: Use a variable here instead of hex format

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]

**Review by volodymyr-soltys97 — APPROVED** [review]
Well done 🔥

**Pull request closed without merging**